app/src/main/java/mozilla/lockbox/LockboxApplication.kt

```diff
@@ -6,15 +6,24 @@
 
 package mozilla.lockbox
 
+import android.annotation.SuppressLint
 import android.app.Application
+import io.sentry.Sentry
+import io.sentry.android.AndroidSentryClientFactory
 import mozilla.components.support.base.log.Log
 import mozilla.components.support.base.log.logger.Logger
 import mozilla.components.support.base.log.sink.AndroidLogSink
 
 val log: Logger = Logger("Lockbox")
 class LockboxApplication : Application() {
+    @SuppressLint("AuthLeak")
```

Comment:
is this actually OK?

Reply:
Eh.. no, it's not great. I will see about turning it into an environment variable.